x-pack/test/functional/page_objects/endpoint_page.ts

```diff
@@ -58,5 +59,13 @@ export function EndpointPageProvider({ getService }: FtrProviderContext) {
             )
         );
     },
+
+    async waitForTableToHaveData(dataTestSubj: string) {
+      await retry.waitForWithTimeout('table to have data', 2000, async () => {
+        const tableData = await this.getEndpointAppTableData(dataTestSubj);
+        if (tableData[1][0] === 'No items found') return false;
+        return true;
```

Comment:
Nit - I'm wondering if the 'No items found' will always be in that part of the table.

Reply:
Yes, it is always part of the endpoint management tables.